src/components/graph/NodeTooltip.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div
+    v-if="tooltipText"
+    ref="tooltipRef"
+    class="node-tooltip"
+    :style="{ left, top }"
+  >
+    {{ tooltipText }}
+  </div>
+</template>
```

Comment:
PrimeVue has [tooltip directive](https://primevue.org/tooltip/) and [popover component](https://primevue.org/tooltip/) available. Maybe we can try take advantage of existing component next time.

Reply:
I looked at those and neither seem particularly useful as the tooltip doesnt seem to have a documented API for specifing a position & showing it, and the popover wraps an element which would mean I need to position the tooltip in the correct position anyway.

Reply:
Popover's show method accept an event and uses the event target's location to position the overlay. We can simulate an event to use it. Tooltip directive might be more tricky to utilize as it requires HTML DOM element to anchor to, and creating so many invisible DOM elements overlay might cause peef issue.

Since you already have a functioning impl ready, I don't think we need to bend ourself to any of the solution above-mentioned. Just for next time, we should access what is already available. PrimeVue is pretty flexible. If passtheough is not enough, you can even inherit the component to get more freedom like I did for Autocomplete and Tree under PrimeVue override for.